helix-term/src/ui/mod.rs

```diff
@@ -284,6 +284,27 @@ pub mod completers {
         })
     }
 
+    pub fn language(editor: &Editor, input: &str) -> Vec<Completion> {
+        let matcher = Matcher::default();
+
+        let mut matches: Vec<_> = editor
+            .syn_loader
+            .language_configs()
+            .filter_map(|config| {
+                matcher
+                    .fuzzy_match(&config.language_id, input)
+                    .map(|score| (config.language_id.clone(), score))
```

Comment:
Could you do the clone on line 304 instead? That way you're not cloning entries that don't match.

Reply:
done.